diff --git a/data/meson.build b/data/meson.build
index e436efe..d9353bd 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -3,7 +3,7 @@ iconsdir = join_paths(datadir, 'icons', 'hicolor')
 appstream_util = find_program('appstream-util', required: false)
 if appstream_util.found()
   test('Validate appstream file', appstream_util, args: [
-    'validate-relax',
+    'validate-relax', '--nonet',
     join_paths(meson.current_source_dir(), 'com.github.rafostar.Clapper.metainfo.xml')
   ])
 endif